site/docs/v1/tech/tutorials/gating/_common-questions-gating.adoc

```diff
@@ -0,0 +1,22 @@
+=== Retroactive Emails Once Gate Enabled?
+
+**Question**: Will all my existing users get emails for either gate once enabled?
+
+**Answer**:  No. More specifically, this is an accurate statement if you have **never** enabled email verification and/or application registration verification and imported your users with a `verified: true` value (as applicable).
+
+To double-check your users, click on [breadcrumb]#Users -> Search Bar# and use this Elasticsearch query string: `verified: false`. Note, this will return if either `user` or `user.registrations[x]` is unverified. Users found by this query will be required to verify either their email or application registration once the gate is enabled. However, **they will not be emailed until they log in**. During the login process, they will be required to click on a link to have a verification email sent to them.
+
+=== Are Manual Changes Possible?
+
+**Question**: What if I have accidentally/inadvertently imported users with a `verified: false` value (either on the user or application registration)?  In other words, I need all my existing users to have a `verified: true` value.
+
+**Answer**: Currently, there is no way to change this value manually. However, there is an open issue adding this functionality. Please upvote if you believe this is needed.
+
+https://github.com/FusionAuth/fusionauth-issues/issues/1319
+
+=== What About Enabling Both Gates?
+
+**Question**: What does it mean to enable both registration and email gates and verification paths?
+
+**Answer**: If you enable both registration verification and email verification, there will be a bit of redundancy (this is fine and expected). Consider the use case of ten applications and one common tenant. Maybe nine of those applications require only email verification, but the tenth application requires an application gate to force user opt-in behaviors.
```

Comment:
You don't answer the question directly here. 

Will the user get 2 emails for the 10th application? (I am guessing yes.)